src/components/team-member-form/TeamMemberForm.tsx

```diff
@@ -120,11 +127,11 @@ function TeamMemberForm({
               First Name
               <Input
                 isValid={validity.firstName !== false}
-                maxLength="35"
-                minLength="2"
+                maxLength={35}
+                minLength={2}
                 name="firstName"
                 onBlur={handleBlur}
-                onChange={event => setFirstName(event.target.value)}
+                onChange={(event): void => setFirstName(event.target.value)}
```

Comment:
I'd suggest to add the event type here and on following occasions. @RobbinHabermehl Same question as above!

Reply:
changed to:

onChange={({target}: React.ChangeEvent<HTMLInputElement>): void => setFirstName(target.value)}